peer/peerwg/ssh.go

```diff
@@ -0,0 +1,38 @@
+package peerwg
+
+import (
+	"context"
+	"net"
+
+	"golang.org/x/crypto/ssh"
+	"golang.org/x/xerrors"
+	"inet.af/netaddr"
+)
+
+func (n *Network) SSH(ctx context.Context, ip netaddr.IP) (net.Conn, error) {
```

Comment:
This is fine for now, but we should probably keep this in the agent package eventually to separate concerns of networking and handshakes over our specific protocol implementations.

Reply:
True